libres/lib/enkf/local_ministep.cpp

```diff
@@ -177,6 +179,30 @@ RowScaling *get_or_create_row_scaling(py::handle obj, const std::string &name) {
 RES_LIB_SUBMODULE("local.ministep", m) {
     using namespace py::literals;
 
+    auto get_obs_active_list_impl = [](py::handle self) -> py::dict {
+        auto ministep = reinterpret_cast<local_ministep_type *>(
+            PyLong_AsVoidPtr(self.attr("_BaseCClass__c_pointer").ptr()));
```

Comment:
use 
```suggestion
        ert::from_cwrap<local_ministep_type>(self);
```
?

Reply:
That was written before `ert::from_cwrap`. Will fix